crates/sui-types/src/crypto.rs

```diff
@@ -301,11 +198,11 @@ impl Signature {
     }
 
     pub fn signature_bytes(&self) -> &[u8] {
-        &self.0[..ed25519_dalek::SIGNATURE_LENGTH]
+        &self.0[..AccountSignature::LENGTH]
```

Comment:
Note that in crypto agility we cannot have a global fixed length (each scheme has (might have) different size)